xml/registry.rnc

```diff
@@ -51,6 +52,17 @@ Type = element type {
     text
 }
 
+# <kinds> defines descriptions for kinds.
+Kinds = element kinds {
+    Kind *
+}
+
+# <kind> defines a description for a single kind.
+Kind = element kind {
+    Name ,
+    attribute desc
+}
```

Comment:
I think `Name` should also be marked as an attribute.

Also, it doesn't make sense to me to define `kinds` and `kind` separately. I think this is how you combine them (tho I also didn't run tests):
```suggestion
# <kinds> defines descriptions for kinds
Kinds = element kinds {
    element kind {
        attribute name ,
        attribute desc
    } *
}
```
(also removed the dot at the end of a `#` comment, to make it more uniform with other comments)

Reply:
`Name` is defined at the end as it's own attribute, I agree that it's somewhat confusing, but it's what most parts of the document do.

I guess I could combine the two definitions, but I tried to follow the structure of the rest of the document.